.lando.extras.yml

```diff
@@ -3,9 +3,9 @@ cypress:
   template:
     services:
       cypress:
-        type: node:20
+        type: node:18
         overrides:
-          image: "cypress/base:20"
+          image: "cypress/base:18"
```

Comment:
Didn't we determine this needed to be a specific version b/c `cypress/base:18` doesn't exist?

Reply:
Oh! `cypress/base:18.16.1`, right?

Reply:
Changed as suggested.